packages/lesswrong/server/callbacks/rateLimits.ts

```diff
@@ -51,6 +52,16 @@ async function enforcePostRateLimit (user: DbUser) {
   if (userIsAdmin(user) || userIsMemberOf(user, "sunshineRegiment") || userIsMemberOf(user, "canBypassPostRateLimit"))
     return;
   
+  const moderatorRateLimit = await getModeratorRateLimit(user)
+
+  const hours = getTimeframeForRateLimit(moderatorRateLimit?.type as RateLimitType)
+
+  const postsInPastTimeframe = await userNumberOfItemsInPastTimeframe(user, Posts, hours)
+
+  if (postsInPastTimeframe > 0 && moderatorRateLimit) {
```

Comment:
Probably should leave a comment documenting that the `postsInPastTimeframe > 0` condition makes sense because all of the rate limits are "max 1 per [time period]", so there isn't (yet) a situation where we'd want to allow a user with >0 of the relevant items to make another one.  But that might change in the future.